Makefile

```diff
@@ -71,7 +71,7 @@ gen-files:
 
 $(BINDIR)/deepcopy-gen: vendor
 	$(DOCKER_GO_BUILD) \
-		sh -c 'go build -o $@ $(LIBCALICO-GO_PKG)/vendor/k8s.io/code-generator/cmd/deepcopy-gen'
+		sh -c 'go build -mod=vendor -o $@ $(LIBCALICO-GO_PKG)/vendor/k8s.io/code-generator/cmd/deepcopy-gen'
```

Comment:
When I delete the generated files and then run `make gen-files` it doesn't recreate them for me.  I suspect that this might have something to do with the fact that `PACKAGE_NAME` is `projectcalico/libcalico-go`, but the `--bounding-dirs` is `github.com/projectcalico/libcalico-go`

Reply:
Great catch. The problem turned to be our version of deepcopy-gen still needing /go/src/github.com/... Fixed now.